### PR TITLE
Listing page react

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Added:
Static design of Product Listing page

### CODE REVIEW

File: src/components/listing/ListingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            discountPercent: 10,
+            closeActive: false
+          }}
+        />
+      </div>
+    </main>
+  );
+};
+
+export default ListingPage;
```
Comment (deekshithmd) [inline src/components/listing/ListingPage.jsx:113]: Code looks clean and good to merge

**Review by deekshithmd — APPROVED** [review]

File: src/components/cart/CartTotalBox/CartTotalBox.jsx [hunk 2]
```diff
@@ -2,32 +2,32 @@ import React from "react";
...
+const CartTotalBox = ({item}) => {
   return (
     <div class="total-box">
       <div class="cart-total-box">
         <p class="p-large text-left text-bold-weight">Price Details</p>
         <hr />
         <div class="side-ways-cart">
-          <p class="p-large text-left">Price (2 items)</p>
-          <p class="p-large text-left">Rs. 1198</p>
+          <p class="p-large text-left">Price ({item.quantity} items)</p>
```
Comment (BALAJI141031) [inline src/components/cart/CartTotalBox/CartTotalBox.jsx:12]: align-text property is inherit property so no need to give for every child instead you can give to parent directly

File: src/routes/ProductListing.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import ListingPage from "../components/listing/ListingPage";
+import FilterSideBar from "../components/listing/FilterBar/FilterSideBar";
+
+import "./ProductListing.css";
+import { Navbar } from "../components/navbar/Navbar";
+
+const ProductListing = () => {
+  return (
+    <>
+      <Navbar />
```
Comment (BALAJI141031) [inline src/routes/ProductListing.jsx:11]: no need to render navbar here you can outside of all routes

File: src/components/listing/ListingPage.jsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
+              <path d="M96 0C107.5 0 117.4 8.19 119.6 19.51L121.1 32H541.8C562.1 32 578.3 52.25 572.6 72.66L518.6 264.7C514.7 278.5 502.1 288 487.8 288H170.7L179.9 336H488C501.3 336 512 346.7 512 360C512 373.3 501.3 384 488 384H159.1C148.5 384 138.6 375.8 136.4 364.5L76.14 48H24C10.75 48 0 37.25 0 24C0 10.75 10.75 0 24 0H96zM128 464C128 437.5 149.5 416 176 416C202.5 416 224 437.5 224 464C224 490.5 202.5 512 176 512C149.5 512 128 490.5 128 464zM512 464C512 490.5 490.5 512 464 512C437.5 512 416 490.5 416 464C416 437.5 437.5 416 464 416C490.5 416 512 437.5 512 464z" />
+            </svg>
+            Add to Cart
+          </button>
+          <button class="no-bg btn-icon-text-outline">Wishlist</button>
+        </div>
+      </div>
+
+      <div className="products">
```
Comment (BALAJI141031) [inline src/components/listing/ListingPage.jsx:41]: here you can use **map** function no need to write code again and agian

**Pull request merged**